web-registry/src/pages/CreditClassDetails/CreditClassDetailsSimple/CreditClassDetailsSimple.utils.ts

```diff
@@ -5,5 +5,5 @@ export const getCreditClassDisplayName = (
   metadata?: Partial<CreditClassMetadataLD>,
 ) => {
   const name = metadata?.['schema:name'];
-  return name ? `${name} (${creditClassId})` : creditClassId;
+  return name ? name : creditClassId;
```

Comment:
Here's the update to the display name. Let me know if this is what you were thinking.